src/eventcontroller/EventController.ts

```diff
@@ -1,16 +1,34 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import EventObserver from '../eventobserver/EventObserver';
+import EventReporter from '../eventreporter/EventReporter';
 import AudioVideoEventAttributes from './AudioVideoEventAttributes';
 import DeviceEventAttributes from './DeviceEventAttributes';
 import EventName from './EventName';
 import MeetingHistoryState from './MeetingHistoryState';
 
 /**
- * [[EventController]] keeps track of events and notifies audio-video observers.
- * An event describes the success and failure conditions for the meeting session.
+ * [[EventController]] keeps track of a list of event observers and notifies them of SDK events.
+ * An event can describe success or failure of SDK components that make use of this controller.
```

Comment:
I feel like this info makes more sense on `publishEvent`.

Reply:
I think it makes more sense to be on the controller to help contextualize better what the controller is actually doing, if I'm thinking about wanting to use the `EventController` I'd look at the documentation for the interface and see that I can give observers that will get notified by events but I don't really know what those events are unless I looked at the documentation on publishing events but typically builders will be observing rather then publishing.

Reply:
Fair enough. Maybe makes sense to just link the guide here too if we are trying to maximize contextualization :).

Reply:
That's fair, I'll add that in.